src/libraries/System.Text.Json/src/System/Text/Json/JsonHelpers.cs

```diff
@@ -134,5 +135,10 @@ public static bool IsFinite(float value)
             return !(float.IsNaN(value) || float.IsInfinity(value));
 #endif
         }
+
+        public static bool IsValidNumberHandlingValue(JsonNumberHandling handling)
+        {
+            return JsonHelpers.IsInRangeInclusive((int)handling, 0, 7);
```

Comment:
nit: Do this instead, to avoid magic numbers
```suggestion
            return JsonHelpers.IsInRangeInclusive((int)handling, 0, (int)(Strict |  AllowReadingFromString |  WriteAsString | AllowNamedFloatingPointLiterals));
```